be/src/vec/functions/array/function_array_enumerate.cpp

```diff
@@ -56,7 +56,6 @@ class FunctionArrayEnumerate : public IFunction {
     static constexpr auto name = "array_enumerate";
     static FunctionPtr create() { return std::make_shared<FunctionArrayEnumerate>(); }
     String get_name() const override { return name; }
-    bool use_default_implementation_for_nulls() const override { return true; }
```

Comment:
do not modify code like this if this pr just wants to do refactor on `check_set_nullable`